### PR TITLE
GB-3975: Rename names from resolver to user defined functions

### PR DESCRIPTION
**Pull request by tomasol**
# Description

Rename resolver occurrences to udf / user-defined function

# Type of change

- [ ] 💔 Breaking
- [ ] 🚀 Feature
- [ ] 🐛 Fix
- [ ] 🛠️ Tooling
- [x] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

File: cli/crates/server/src/bridge/errors.rs [hunk 1]
```diff
@@ -21,11 +21,11 @@ pub enum ApiError {
     SqlError(#[from] SqlxError),
     #[error("server error")]
     ServerError,
-    #[error("resolver {0} is invalid")]
-    ResolverInvalid(String),
+    #[error("user-defined function invocation error")]
+    UdfInvocation,
```
Comment (Finistere) [inline cli/crates/server/src/bridge/errors.rs:25]: Is the name of the UDF useless?

**Review by Finistere — APPROVED** [review]

**Comment by yoav-lavi** [comment]
@tomasol why are we making this change? What do we gain by calling it UDF and not resolver?

File: cli/crates/server/src/bridge/errors.rs (hunk 1 above)
Comment (Finistere) [reply cli/crates/server/src/bridge/errors.rs:25]: Nice, thx!

**Comment by tomasol** [comment]
@yoav-lavi It is a preparation for https://linear.app/grafbase/issue/GB-3977/implement-authorizer-custom-auth-function

**Pull request merged**